hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java

```diff
@@ -1314,7 +1314,7 @@ public static Put makePutFromRegionInfo(RegionInfo regionInfo, long ts) throws I
   /**
    * Generates and returns a Delete containing the region info for the catalog table
    */
-  private static Delete makeDeleteFromRegionInfo(RegionInfo regionInfo, long ts) {
+  public static Delete makeDeleteFromRegionInfo(RegionInfo regionInfo, long ts) {
```

Comment:
Is it like this on master branch?

This seems extreme for a test? We have to do this?

Reply:
This was part of Duo's https://issues.apache.org/jira/browse/HBASE-24633, checked into master but not back ported.

Reply:
I don't see this change over in https://github.com/apache/hbase/pull/2003/files

There are conflicts w/ this patch?

Thanks @clarax

Reply:
Not sure what that PR was not in this branch but there shouldn't be conflicts.